CMSgov/mint-app
# [Easi 4290]additional translation types be

# EASI-4290

## Changes and Description

- Added translations for additional types
   - plan_discussion
   - discussion_reply
   - plan_cr
   - plan_tdl
   - plan_document
   - operational_need
   - operational_solution
   - operational_solution_subtask
 - Change pluralization of objects to match backend and database naming conventions 
- Update method calls to fetch the user account service to also return an error
   - This enables us to handle errors that are caused if the User Account Service isn't on the context (EG, there is no data loader)



## How to test this change
1. Verify tests pass for the new types
2. Verity that the user account unit tests also pass as expected
3. Translate seeded data.
   a. Note, some translations will need to be adjusted to handle edge cases (for example, we probably don't need to show an operational need was created at model plan creation. Task list sections might also be handled differently, so they don't show on insert.
     


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
